e2e-tests/ukef/cypress/support/portal-api/createGefFacilities.js

```diff
@@ -0,0 +1,21 @@
+const { logIn, createGefFacilities, updateGefFacilities } = require('./api');
+const { getIdFromNumberGenerator } = require('../external-api/api');
+
+module.exports = (dealId, facilities, user) => {
+  console.info('createGEfFacilities::');
+
```

Comment:
Is this console.info a pattern in other similar files? If we want to keep it, the `E` should be lowercase
```suggestion
  console.info('createGefFacilities::');
```

Reply:
I think we can remove this.